src/main/java/monitoringsystemturbo/Main.java

```diff
@@ -23,6 +22,10 @@
     public void start(Stage primaryStage) throws Exception {
         primaryStage.setTitle("MonitoringSystemTurbo");
 
+        if (!Startup.isAppInStartup() && Startup.isAppJar()) {
```

Comment:
consider moving this concatenated condition to a method with a name indicating what is checked. Readability will grow

Reply:
wydaje mi się, że właśnie czytelność by się zmniejszyła, bo na jaka nazwę bym mógł zmienić?
w nazwie musiałbym dodać and, co oznacza już złamanie SRP

Reply:
ok, do zasady SRP czasem trzeba podchodzić z pewnym dystansem, choć też uważam ją za taką, którą warto się kierować. Rzeczywiście metoda z 'and' w nazwie zwykle nie wróży nic dobrego, więc może zostać tak jak jest